test_treematcher.py

```diff
@@ -116,96 +120,78 @@ def test_lineages(self):
                                   \-Any primate taxid except human
 
         """
-        t1 = PhyloTree("(9601.ENSPPYP00000022176:1,9593.ENSGGOP00000009720:1);", format=1, quoted_node_names=False)
-        t2 = PhyloTree(" (9361.ENSDNOP00000016844:1,9258.ENSOANP00000032529:1);", format=1, quoted_node_names=False)
```

Comment:
Some trees, like t2 were expected to provide 0 matches.. I fixed the test to consider that instead of just failing